### PR TITLE
[PAN-44] BQPump tweaks for Panalyt

### PR DESCRIPTION
**Pull request by soracte**
This change is [Reviewable](https://reviewable.io/reviews/doctusoft/bqpump/6)

### CODE REVIEW

File: bqpump-core/src/main/java/ai/aliz/bqpump/notification/PubsubUtil.java [hunk 1]
```diff
@@ -75,21 +71,6 @@ public void deleteTopic(ProjectTopicName topicName) {
         }
     }
     
-    @SneakyThrows
-    public void setPolicyForTopic(String projectId, TopicName topicName, String role, String members) {
```
Comment (soracte) [inline bqpump-core/src/main/java/ai/aliz/bqpump/notification/PubsubUtil.java]: This was called when creating the bucket upload notification only. However, I found that setting up the notification works just fine without calling this method. Could you please explain why it was called @csabakassai @zkauker ?

**Comment by csabakassai** [comment]
COnflict

File: bqpump-shared/src/main/java/ai/aliz/bqpump/filter/VersionToDatastoreNamespaceFilter.java [hunk 2]
```diff
@@ -51,8 +51,10 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
             NamespaceManager.set(null);
             VersionToNamespace versionToNamespace = ObjectifyService.ofy().load().type(VersionToNamespace.class).id(versionId).now();
             if (versionToNamespace != null) {
+                log.info("Using namespace " + versionToNamespace.getNamespace() + "...");
```
[the remark this answers is not in the document]
Comment (nbali) [reply bqpump-shared/src/main/java/ai/aliz/bqpump/filter/VersionToDatastoreNamespaceFilter.java]: (in theory this makes sense, in reality I never understood this... you will never turn off INFO logging...)

File: bqpump-core/src/main/java/ai/aliz/bqpump/task/BigqueryQueryDeferredTask.java [hunk 3]
```diff
@@ -191,4 +183,25 @@ private FormatOptions buildOptionsForFileFormat(String fileUrl) {
         throw new IllegalArgumentException(String.format("The name of the uploaded file is invalid: %s" +
                                                                  "It should end with .json or .csv according to it's content type.", fileUrl));
     }
+    
+    private String getQueryFromConfig() {
+        String template;
+        
+        if (!StringUtils.isBlank(queryTemplate)) {
```
[the remark this answers is not in the document]
Comment (nbali) [reply bqpump-core/src/main/java/ai/aliz/bqpump/task/BigqueryQueryDeferredTask.java]: 'queryPath' would be a wrong name then

File: bqpump-core/src/main/java/ai/aliz/bqpump/notification/PubsubUtil.java (hunk 1 above)
Comment (soracte) [reply bqpump-core/src/main/java/ai/aliz/bqpump/notification/PubsubUtil.java]: okay I'll try again with the role given to the service account

**Pull request closed without merging**